chain/client/src/rocksdb_metrics.rs

```diff
@@ -115,11 +132,11 @@ impl RocksDBMetrics {
 }
 
 fn get_prometheus_metric_name(stat_name: &str) -> String {
-    format!("near_{}", stat_name.replace('.', "_"))
+    format!("near_{}", stat_name.replace(".", "_").replace("-", "_"))
 }
 
 fn get_metric_name_summary_count_gauge(stat_name: &str) -> String {
-    format!("near_{}_count", stat_name.replace('.', "_"))
+    format!("near_{}", stat_name.replace(&['.', '-'], "_"))
```

Comment:
Why remove `_count` suffix? I don’t think this function needs to be changed at all.